test/services/depreciation_calculator_test.rb

```diff
@@ -0,0 +1,56 @@
+require 'test_helper'
+
+class FinancialYearCloseTest < ActiveSupport::TestCase
```

Comment:
The class doesn't match the filename here.

Reply:
oops